subsys/bluetooth/Kconfig

```diff
@@ -83,6 +83,34 @@ config BT_BROADCASTER
 
 endmenu
 
+config BT_ADV_EXT
+	bool "Extended Advertising and Scanning support"
+	help
+	  Select this to enable Extended advertising API support.
+	  This enable multiple advertising sets with extended advertising
+	  data and advertising with long range support.
+	  Enables extended scanner with support for receiving extended
+	  advertising data and scanning with long support.
+	  Enables the ability create connection using the long range feature.
+
+if BT_ADV_EXT
+config BT_ADV_EXT_LEGACY_SUPPORT
+	bool "Support starting advertising through legacy commands"
+	help
+	  Select this to enable the use of the Legacy Advertising HCI commands.
+	  This option should be used where the capabilities of the controller
+	  is not known.
+
+config BT_MAX_ADV_SET
```

Comment:
I know we have `BT_MAX_CONN` (probably not worth fixing anymore), but it'd be nice if everything inside this `if BT_ADV_EXT` conditional block was prefixed with `BT_ADV_` or perhaps even with `BT_ADV_EXT_`

Reply:
Makes sense, I will update the ADV_SET (and leave BT_MAX_CONN alone).